validator/sawtooth_validator/journal/consensus/dev_mode/dev_mode_consensus.py

```diff
@@ -81,23 +144,44 @@ def __init__(self, block_cache, state_view):
     def verify_block(self, block_state):
         return True
 
+    def compute_block_weight(self, block_state):
+        # longest chain wins
+        return block_state.block_num
+
 
 class ForkResolver(ForkResolverInterface):
-    """ DevMode ForkResolver implementation
-    """
+    # Provides the fork resolution interface for the BlockValidator to use
+    # when deciding between 2 forks.
     def __init__(self, block_cache):
         super().__init__(block_cache)
 
         self._block_cache = block_cache
 
     def compare_forks(self, cur_fork_head, new_fork_head):
         """
+
```

Comment:
Add a comment that describes the algorithm. The longest chain is selected and the lowest hash value of the previous block id and publisher signature is used as a tie breaker.